packages/augur-ui/src/modules/modal/add-funds.tsx

```diff
@@ -58,42 +51,41 @@ const addFundsTorus = async (amount, address) => {
 
 export const AddFunds = ({
   autoSelect = false,
-  tokenToAdd = DAI,
 }: AddFundsProps) => {
+  const BUY_MIN = 20;
+  const BUY_MAX = 250;
   const { loginAccount, modal, actions: {closeModal} } = useAppStatusStore();
-  const isRelayDown = false;
-  const { useSigner, initialSwapToken, initialAddFundsFlow } = modal;
-  const showTransfer = modal.showTransfer || false;
+  const { env: config, ethToDaiRate, repToDaiRate } = useAppStatusStore();
```

Comment:
these should all be in the same const definition, not sure why we have 2 calls to `useAppStatusStore()`